Update DFS.mdx (#2240)
cpinitiative/usaco-guide: content/3_Silver/DFS.mdx

> explore other parts of the graph. The algorithm keeps track of visited nodes,
> so that it processes each node only once.

> DFS is useful when working with Graph problems. Most Silver problems involving graphs can be solved using DFS. This is why it is recommended to commit the strategies presented below to heart. 

Reviewer: ```suggestion
DFS is very useful when doing graph problems. Most silver problems involving graphs can be solved using DFS. This is why we recommended committing the strategies presented below to heart. 
```

Reviewer: ^ just some wording changes
